src/Features/Core/Portable/Completion/Providers/SymbolCompletionItem.cs

```diff
@@ -288,6 +314,28 @@ public static string GetInsertionText(CompletionItem item)
                 filterText, supportedPlatforms, properties, tags);
         }
 
+        public static CompletionItem CreateWithSymbolId(
```

Comment:
This is a new overload, but there are already two overloads (with default parameters), so this is probably problematic.
I created a new overload because I wanted to insert the new parameter in the middle of the parameter list and didn't want to break any positional callers:

* `string displayTextPrefix` between `string displayTextSuffix` and `IReadOnlyList<ISymbol> symbols`
* `Glyph? glyph`between `string filterText` and `SupportedPlatformData supportedPlatforms`
 
### Exisiting

https://github.com/dotnet/roslyn/blob/dcc03e7d083c7950a4864d9343938047420624bf/src/Features/Core/Portable/Completion/Providers/SymbolCompletionItem.cs#L271-L282

https://github.com/dotnet/roslyn/blob/dcc03e7d083c7950a4864d9343938047420624bf/src/Features/Core/Portable/Completion/Providers/SymbolCompletionItem.cs#L298-L309

### New

https://github.com/dotnet/roslyn/blob/dcc03e7d083c7950a4864d9343938047420624bf/src/Features/Core/Portable/Completion/Providers/SymbolCompletionItem.cs#L317-L330

Should I merge the new overload into the existing one (not marked with the COMPAT comment)?

Reply:
I integrated my new overload into the existing overload, which can be changed (no COMPAT comment) in 1455039.

Reply:
>  I wanted to insert the new parameter in the middle of the parameter list and didn't want to break any positional callers:

These are internal APIs that only roslyn can use.  So better to just change teh single member and update callers as appropriate.

Reply:
Done: 14550391